README.md

```diff
@@ -86,5 +91,7 @@ The most common mood is :-)
 ```
 
 **think**: Should you use an array or a hash to solve this problem? Why?
+For counting until happy days, an Array because order matters
+For overall mood, a Hash because order does not matter. But I also already have an array to work with so I just recycled the data I already had because honestly it's past midnight and my brain is only working partially. 
```

Comment:
Zzzz. Sleepy. 

You were correct in thinking a Hash was the answer, though your reasoning is incorrect. It doesn't have to do with the ordering (or lack of). It has to do with the fact that we want to count the occurrences of an indeterminate number of things, which is often solved utilizing the key/value pairings of a hash. (This is common in whiteboarding problems too)